GTG/gtk/data/context_menus.ui

```diff
@@ -5,13 +5,13 @@
   <menu id="closed_task_menu">
     <section>
       <item>
-        <attribute name="label" translatable="yes">Edit</attribute>
+        <attribute name="label" translatable="yes">Edit...</attribute>
```

Comment:
Unicode typography for ellipsis - https://developer.gnome.org/hig/stable/typography.html.en